src/Directory.Packages.props

```diff
@@ -25,7 +25,8 @@
     <PackageVersion Include="Microsoft.TestPlatform.ObjectModel" Version="17.2.0" />
     <PackageVersion Include="Microsoft.TestPlatform.Portable" Version="17.2.0" />
     <PackageVersion Include="Microsoft.TestPlatform.TranslationLayer" Version="17.2.0" />
-    <PackageVersion Include="System.IO.Abstractions" Version="17.2.26" />
+    <PackageVersion Include="TestableIO.System.IO.Abstractions" Version="19.1.5" />
+    <PackageVersion Include="TestableIO.System.IO.Abstractions.Wrappers" Version="19.1.5" /> 
```

Comment:
Why do we suddenly need this package?

Reply:
The one we used (System.IO.Abstractions) referenced both TestableIO's `Abstractions` and `Abstractions.Wrappers`. Without `Wrappers`, the build fails.

Reply:
`Abstractions` contains the `IFileSystem` interface, `Abstractions.Wrappers` contains the `FileSystem` class.

Reply:
`Abstractions.Wrappers` references `Abstractions`, so I can leave the latter out, good catch.